cfme/common/__init__.py

```diff
@@ -85,6 +86,103 @@ def _assign_unassign_policy_profiles(self, assign, *policy_profile_names):
         details_view = self.create_view(navigator.get_class(self, 'Details').VIEW)
         details_view.flash.assert_no_error()
 
+    def assign_policy_profiles_multiple_entities(self, entities, conditions, *policy_profile_names):
```

Comment:
Is this providing anything unique from the existing assign_policy_profile methods above, other than supporting multiple entities?

I'm hesitant to duplicate most/all of the actions for the existing methods, and would prefer modifying them to support passed entities.

Now, this method is on the PolicyProfileAssignable class, and the docblock for it specifies the intent is for anything implementing a load_details method.

I think if we want separate methods for collection support, a separate class should be created for use as a BaseCollection mixin, instead of a BaseEntity mixin.

Reply:
This method support multiple entities, which means that its different from assign_policy_profile also by the view that we navigate from.
On this method we check functionality from collection\ALL view, which is different from Details view.
Also because of collection view we use the apply method to check entities , which is not needed at all on Details view.
I think that we need here separate methods.

Reply:
I believe I understand all of the differences between the two approaches then:
* navigation (collection vs entity)
* widget interaction (checked + plural dropdown action vs details+singular dropdown action)

These could both be captured by a method which handles a list of entities, or singular, and applies a `from_collection` boolean argument to determine through which screen to take action.

That being said, its a complex refactor and API change for this mixin, and one that's definitely outside the scope of this PR.

I suggest, in the future, to put changes like these methods (especially for anything in `cfme/common/`) in a separate PR for individual review.  I understand the difficulty sometimes in splitting out these pieces of logic from your branch for test or container framework changes, but they should make your review experience a bit better. I myself need to do that with #6538 because the scope has grown out of control. Separate PRs for change to core framework components are far easier to review and discuss the framework impacts and intended use cases.  

I'm happy to help you git (get it??? :P) setup to cherry-pick or rebase your branches on things other than master or to pull in changes from local or remote branches.